.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "globals": {
+        "Atomics": "readonly",
```

Comment:
If I'm reading this correctly, I don't think this is needed since it isn't referenced anywhere in the module

Reply:
FWIW, this content and the item below are added automatically as part of `eslint --init` boilerplate. I can remove them if you like, however.

Reply:
No worries. I thought it might have been a copy/paste error